chat-plugins/scavengers.js

```diff
@@ -26,6 +28,42 @@ if (scavengersRoom) {
 	}
 }
 
+let scavengersLB;
+let dataFile = './config/chat-plugins/scavdata.json';
+fs.readFile(dataFile, 'utf8', function (err, contents) {
+	if (err) {
+		scavengersLB = {};
+		return;
+	}
+	scavengersLB = JSON.parse(contents);
```

Comment:
you probably still want to make sure this doesn't crash in case of malformed json.